add articles about sending and receiving with bridge (#137)
svix/svix-docs: docs/receiving/verifying-payloads/receiving-with-bridge.mdx

---
title: Receiving Webhooks with Bridge
---

import CodeTabs from '@theme/CodeTabs';
import TabItem from '@theme/TabItem';

> [Svix Bridge] can be useful in cases where you want to consume webhooks and write the payloads to a message queue.

Reviewer: ```suggestion
[Svix Bridge] is useful in cases where you want to consume webhooks and write the payloads to a message queue.
```
